azure-samples/src/test/java/com/microsoft/azure/management/samples/ComputeSampleTests.java

```diff
@@ -135,7 +135,11 @@ public void testManageManagedDisks() {
 
     @Test
     public void testManageContainerServiceUsingKubernetes() {
-        Assert.assertTrue(ManageContainerServiceUsingKubernetes.runSample(azure));
+        if (IS_MOCKED) {
```

Comment:
I think this is fine - as long as the `IS_MOCKED` check happens only in `*Testss.java`.